Rules.md

```diff
@@ -133,9 +133,10 @@ player available, all write down their action and execute that action.
   previous one concluded.
 
 * **Train units**:<br>
-    * In regions with a strategic city: +2 infantry or +1 artillery or upgrade 2 infantry to artillery
-    * In other land regions: +1 infantry or upgrade 1 infantry to artillery
-    * In sea regions: Nothing happens
+  You can add/upgrade units in this region up to an additional cost of 2 if it's a strategic city,
+  or 1 if it's a normal land region.
```

Comment:
Was worried that adding 'coin' would have people think you need to pay for training.
But with an example, this might be fine.